src/euphorie/client/browser/country.py

```diff
@@ -382,17 +382,50 @@ def form_action_name(self):
     def template(self):
         return self.index
 
+    @property
+    def organisation_options(self):
+        organisation_view = api.content.get_view(
+            "organisation", self.context, self.request
+        )
+        return [
+            {
+                "label": organisation.title,
+                "value": organisation.owner_id,
+                "selected": "selected"
+                if organisation.owner_id == int(self.request.get("organisation", ""))
```

Comment:
I would be more conservative, `int("")` will raise a `ValueError`, what about:
```suggestion
                if str(organisation.owner_id) == self.request.get("organisation", "")
```
it appears safer to me.

Reply:
Good point! Let's do that.